src/main/java/com/reckue/post/utils/Check.java

```diff
@@ -0,0 +1,21 @@
+package com.reckue.post.utils;
```

Comment:
Move new classes from`src/main` to `src/test`.

Reply:
I can't do it because the annotations of Lombok doesn't work at these folders.  I've already tried.

Reply:
Then use without a lombok. Because this class will not be transferred to production after testing.
It is needed only in testing.

Reply:
testCompileOnly 'org.projectlombok:lombok:1.18.12'
testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'

Reply:
@camelya58 please try change scope of the lombok dependency.

Reply:
More info: https://projectlombok.org/setup/gradle

```gradle
compileOnly 'org.projectlombok:lombok:1.18.12'
annotationProcessor 'org.projectlombok:lombok:1.18.12'

testCompileOnly 'org.projectlombok:lombok:1.18.12'
testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
```